docs/reference/site-config.md

```diff
@@ -426,7 +426,7 @@ When set to `true`, the production app will be built in [MPA Mode](../guide/mpa-
 
 ### appearance
 
-- Type: `boolean | 'dark' | import('@vueuse/core').UseDarkOptions`
+- Type: `boolean | 'dark' | 'force-dark' | import('@vueuse/core').UseDarkOptions`
```

Comment:
Shouldn't this be in description below as well? Or is it intended not to show there.

Reply:
I don't think anyone should have a confusion with what `force-something` means, but yeah we can add that in the docs.